--- a/src/main/java/org/bukkit/craftbukkit/Main.java
+++ b/src/main/java/org/bukkit/craftbukkit/Main.java
@@ -139,6 +_,14 @@
                         .defaultsTo(new File[] {})
                         .describedAs("Plugin directory");
 
+                // Mirage start - Mirage config files
+                this.acceptsAll(asList("mirage", "mirage-settings", "purpur", "purpur-settings"), "File for mirage settings")
+                    .withRequiredArg()
+                    .ofType(File.class)
+                    .defaultsTo(new File("mirage.yml"))
+                    .describedAs("Yml file");
+                // Mirage end - Mirage config files
+
                 this.accepts("server-name", "Name of the server")
                         .withRequiredArg()
                         .ofType(String.class)
